src/scenic/syntax/translator.py

```diff
@@ -1824,6 +2074,40 @@ def visit_Call(self, node):
 		"""
 		func = node.func
 		newArgs = []
+
+		# wrap temporal requirement in lambda
+		if isinstance(func, Name) and func.id in PROPOSITION_FACTORY:
+			if not self.canUseTemporalOps:
+				self.parseError(node, f"Operator {func.id} is not allowed in this statement")
+			checkedArgs = node.args
+			self.validateSimpleCall(node, (1, None), args=checkedArgs)
```

Comment:
You can just leave off the `args` keyword argument here and drop `checkedArgs`.

Reply:
Thanks! Fixed in 93f296c321882bb64f72387900eeaf5029e95ce2